docs/es/week13/13-3.md

```diff
@@ -0,0 +1,386 @@
+
+<!-- Finally, we define our `GatedGCN` model which comprises of the previously defined classes: `GatedGCN_layer` and `MLP_layer`. The definition of our model (`GatedGCN`) is shown below. -->
+Finalmente, definimos nuestro modelo `GatedGCN` que contiene las clases anteriormente mencionadas: `GatedGCN_layer` y `MLP_layer`. La definición de nuestro modelo (`GatedGCN`) se muestra a continuación.
+
+
+ ```python
+ class GatedGCN(nn.Module):
+    def __init__(self, input_dim, hidden_dim, output_dim, L):
+        super().__init__()
+        self.embedding_h = nn.Linear(input_dim, hidden_dim)
+        self.embedding_e = nn.Linear(1, hidden_dim)
+        self.GatedGCN_layers = nn.ModuleList([
+            GatedGCN_layer(hidden_dim, hidden_dim) for _ in range(L)
+        ])
+        self.MLP_layer = MLP_layer(hidden_dim, output_dim)
+    def forward(self, g, h, e, snorm_n, snorm_e):
+        # embedding de entrada
+        h = self.embedding_h(h)
+        e = self.embedding_e(e)
+        # capas convolucionales del grafo
+        for GGCN_layer in self.GatedGCN_layers:
+            h, e = GGCN_layer(g, h, e, snorm_n, snorm_e)
+        # clasificador MLP
+        g.ndata['h'] = h
+        y = dgl.mean_nodes(g,'h')
+        y = self.MLP_layer(y)
+        return y
+ ```
+
+<!-- In our constructor, we define the embeddings for `e` and `h` (`self.embedding_e ` and `self.embedding_h`), `self.GatedGCN_layers` which is list (of size $L$) of our previously defined model: `GatedGCN_layer`, and finally `self.MLP_layer` which was also defined before. Next, using these initializations, we simply foward through the model and output `y`. -->
+En nuestro constructor, definimos los embeddings para `e` y `h` (`self.embedding_e ` y `self.embedding_h`), `self.GatedGCN_layers`, que es una lista (de tamaño $L$) del modelo definido anteriormente: `GatedGCN_layer`, y finalmente `self.MLP_layer`, que también se ha definido anteriormente.
+A continuación, utilizamos estas inicializaciones y simplemente enviamos hacia delante a través del modelo generamos la salida `y`.
+
+<!-- To better understand the model, we initiate an object of the model and print it for better visualization: -->
+Para entender mejor el modelo, inicializamos un objecto del modelo e imprimimos su contenido para una mejor visualización:
+```python
+# instanciamos la red
+model = GatedGCN(input_dim=1, hidden_dim=100, output_dim=8, L=2)
+print(model)
+```
+
+<!-- The main structure of the model is shown below: -->
+La estructura principal del modelo se muestra a continuación:
+
+```python
+GatedGCN(
+  (embedding_h): Linear(in_features=1, out_features=100, bias=True)
+  (embedding_e): Linear(in_features=1, out_features=100, bias=True)
+  (GatedGCN_layers): ModuleList(
+    (0): GatedGCN_layer(...)
+    (1): GatedGCN_layer(... ))
+  (MLP_layer): MLP_layer(
+    (FC_layers): ModuleList(...))
```

Comment:
From here on the comments are marked with red. Were you able to compile it?

Reply:
Yeah I noticed, my blind guess is that because the snippet above is labeled as Python but it's not real Python syntax it's breaking somehow the color highlighting below. The page rendered correctly anyway.
You can see the same issue happening in this PR for the chinese translation at exactly the same point:
[https://github.com/Atcold/pytorch-Deep-Learning/pull/571/files](https://github.com/Atcold/pytorch-Deep-Learning/pull/571/files)

Reply:
Just for reference:
<img width="807" alt="Screen Shot 2020-09-21 at 15 18 47" src="https://user-images.githubusercontent.com/11987768/93737453-3a90ad80-fc1e-11ea-8341-42b750b4606d.png">

Reply:
Great! thank you for your time